listenbrainz/listen.py

```diff
@@ -179,6 +179,8 @@ def to_influx(self, measurement):
             'time' : self.ts_since_epoch,
             'tags' : {
                 'user_name' : escape(self.user_name),
+                'artist_msid' : self.artist_msid,
```

Comment:
Right. Sadly this will bloat our data for little gain, but I can't think of a much better way of doing this.